### PR TITLE
ISSUE #3779 - Fix Storybook components

### LINKED ISSUE #3779: Review stories in storybook
### Description
Some stories are broken. We need to review them and fix where 

### Tasks
- [ ] Check all the stories in storybook
- [ ] FIx the problems that arised

### PR DESCRIPTION
**Pull request by Amantini1997**
This fixes #3779

#### Description
cleaned up stories

### CODE REVIEW

File: frontend/stories/buttons/Button.stories.tsx [hunk 1]
```diff
@@ -57,21 +55,9 @@ export default {
 } as ComponentMeta<typeof Button>;
 
 // More on component templates: https://storybook.js.org/docs/react/writing-stories/introduction#using-args
-const Template: ComponentStory<typeof Button> = (args) => <Button {...args}>Material ui button</Button>;
+const Template: ComponentStory<typeof Button> = (args) => <Button {...args}>Controls Button</Button>;
 
-export const Primary = Template.bind({});
 // More on args: https://storybook.js.org/docs/react/writing-stories/args
-Primary.args = {
```
Comment (The-Daniel) [inline frontend/stories/buttons/Button.stories.tsx:64]: Any reason the 'outlined' examples have been removed?
Also you've removed the 'label' and 'label-outlined' variants from the options, so once you change the variant of one of these you can't change back

File: frontend/stories/outputs/SuccessMessage.stories.tsx [hunk 2]
```diff
@@ -24,19 +24,28 @@ export default {
 		children: {
 			description: 'The text or component to display',
```
Comment (The-Daniel) [inline frontend/stories/outputs/SuccessMessage.stories.tsx:25]: Should this include 'component'?

**Review by The-Daniel — CHANGES_REQUESTED** [review]

**Review by The-Daniel — APPROVED** [review]

**Review by sanmont3drepo — CHANGES_REQUESTED** [review]
There are some icons that doesnt show. I think is because the extension of these are `.svg` instead of `.svg.tsx`, nevertheless I think they should show up in the list:

image

**Comment by sanmont3drepo** [comment]
> There are some icons that doesnt show. I think is because the extension of these are `.svg` instead of `.svg.tsx`, nevertheless I think they should show up in the list:
> 
> image

Ok, re tested it in my computer and it worked 🤷

**Review by The-Daniel — APPROVED** [review]

**Pull request merged**